tests/flow/test_rdb_c_layout.py

```
@@ -0,0 +1,234 @@
+        # without its terminator loaded on C as `XLibte` -- the name ran into
+        # whatever followed it in the RDB -- and none of its functions could be
+        # found. `GRAPH.UDF LIST` still reported a library, so it failed quietly.
+        #
+        # Only the *writing* half needs asserting here. The reading half cannot
+        # regress unnoticed: since we now write the terminator, a reader that stops
+        # stripping it cannot load its own output, and the whole of test_udf fails
+        # to run. Ablating each half separately is what established that.
+        self._reset()
+        lib = "NulTermLib"
+        self.db.udf_load(lib, "function double (x) { return x * 2; }\n"
+                              f"falkor.register ('double', double);", True)
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)